### PR TITLE
Update wording of boolean field descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request updates boolean field descriptions to use 'if' instead of 'whether'.

> 'Whether' is a conjunction that usually starts a subordinate clause that expresses an indirect question involving two stated or implied possibilities or alternatives. 'If' can also be used in this sense.

## Links

[@Boydbueno's input on another PR](https://github.com/Rebilly/api-definitions/pull/1172#discussion_r1050867509)

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ properties:
     type: string
     readOnly: true
   invalidate:
-    description: Specifies whether to invalidate a token after an exchange is performed.
+    description: Specifies if to invalidate a token after an exchange is performed.
```
Comment (Boydbueno) [inline openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml:19]: This one does not sound quite right. 
```suggestion
    description: Specifies whether to invalidate a token after an exchange is performed.
```

Comment (Boydbueno) [reply openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml:19]:
```suggestion
    description: Specifies if a token is invalidated after an exchange is performed.
```

File: openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ allOf:
           28: Prepaid activation and load.
       declineNotEnrolled:
         type: boolean
-        description: Specifies whether to decline transactions if a payment card is not enrolled.
+        description: Specifies if to decline transactions if a payment card is not enrolled.
```
Comment (Boydbueno) [inline openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:76]:
```suggestion
        description: Specifies whether to decline transactions if a payment card is not enrolled.
```

Comment (Boydbueno) [reply openapi/components/schemas/ThreeDSecureServers/ThreeDSecureIO3dsServer.yaml:76]:
```suggestion
        description: Specifies if a transaction is declined if a payment card is not enrolled.
```

**Review by Boydbueno — COMMENTED** [review]
I wasn't sure about two of the changes, please check them out!

File: openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml (hunk 1 above)
Comment (noon-dawg) [reply openapi/components/schemas/CustomerAuthentication/CustomerJWT.yaml:19]: Thanks for spotting these @Boydbueno 🙏

**Review by Boydbueno — APPROVED** [review]

**Review by i-yasko — APPROVED** [review]

**Pull request merged**